maap/dps/dps_job_model.py

```diff
@@ -81,7 +81,7 @@ def submit_job(self):
         request_xml = self.generate_request_xml()
         logger.debug(f'request_xml: {request_xml}')
         response = requests.post(
-            url=ConfigReader().dps_job,
+            url=MaapConfig().dps_job,
```

Comment:
Similar to `DpsJob`, add `maap: MAAP` to the `__init__` method and set `self.maap = maap`, which will then let you do this:
```suggestion
            url=self.maap.config.dps_job,
```